ui/components/app/asset-list/asset-list.js

```diff
@@ -98,19 +123,37 @@ const AssetList = ({ onClickAsset }) => {
       />
       {detectedTokens.length > 0 &&
         !istokenDetectionInactiveOnNonMainnetSupportedNetwork && (
-          <DetectedTokensLink setShowDetectedTokens={setShowDetectedTokens} />
+          <>
+            {process.env.MULTICHAIN ? (
+              <DetectedTokensBanner
+                actionButtonOnClick={() => setShowDetectedTokens(true)}
+                margin={4}
+              />
+            ) : (
+              <DetectedTokensLink
+                setShowDetectedTokens={setShowDetectedTokens}
+              />
+            )}
+          </>
         )}
       <Box marginTop={detectedTokens.length > 0 ? 0 : 4}>
-        <Box justifyContent={JustifyContent.center}>
-          <Typography
-            color={Color.textAlternative}
-            variant={TypographyVariant.H6}
-            fontWeight={FONT_WEIGHT.NORMAL}
-          >
-            {t('missingToken')}
-          </Typography>
-        </Box>
-        <ImportTokenLink />
+        {process.env.MULTICHAIN ? (
+          <MultichainImportTokenLink margin={4} />
+        ) : (
+          <>
+            <Box justifyContent={JustifyContent.center}>
+              <Text
+                color={Color.textAlternative}
+                variant={TextVariant.bodySm}
+                as="h6"
+              >
+                {t('missingToken')}
+              </Text>
+            </Box>
```

Comment:
Do you need this Box wrapper? You could reduce the amount of html and just use `textAlign={TEXT_ALIGN.CENTER}` ?
```suggestion
            <Text
              color={Color.textAlternative}
              variant={TextVariant.bodySm}
              as="h6"
              textAlign={TEXT_ALIGN.CENTER}
            >
              {t('missingToken')}
            </Text>
```